.travis.yml

```diff
@@ -21,15 +21,18 @@ script:
   - cargo audit
 
 before_deploy:
-  - export TRAVIS_TAG=$GIT_TAG
+  - cd git
```

Comment:
Problem with this will be that it needs to publish all of the projects I think to work. I wonder what the approach should be for publishing Cargo workspaces. Some Googling needed I think!